tests/scripts/thread-cert/border_router/MATN_12_HopLimitProcessing.py

```diff
@@ -151,6 +151,7 @@ def test(self):
         self.collect_rloc16s()
         self.collect_rlocs()
         self.collect_extra_vars()
+        self.simulator.go(5)
```

Comment:
It won't be useful adding delay after `collect_*` methods. 

If it's true that `ipaddr rloc` failed to get the command output within the timeout, should we increase the command timeout in simulations?

Reply:
I agree the delay in `collect_*` won't help. However, I don't have much idea how this test case is related to simulation since  `VIRTUAL_TIME` is set to 0 in border router test cases. Could you send a PR when you get a chance?

Reply:
Without `VIRTUAL_TIME`, it's real-time simulation. The default tiemout is 10, which may not be enough in Docker Runners. 
```
self.pexpect = pexpect.popen_spawn.PopenSpawn(cmd, timeout=10)
```

Reply:
Thanks! PR updated.